loadtest/control/utils_test.go

```diff
@@ -6,14 +6,35 @@ package control
 import (
 	"errors"
 	"fmt"
+	"math/rand"
+	"os"
 	"regexp"
+	"strconv"
 	"strings"
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+func TestMain(m *testing.M) {
+	s := os.Getenv("MM_LOADTEST_SEED")
+	var seed int64
+	if s != "" {
+		v, err := strconv.Atoi(s)
+		if err != nil {
+			panic(err)
+		}
+		seed = int64(v)
```

Comment:
This is one case where a *shared* util function would come in handy.

Reply:
Moved it 👍